### PR TITLE
UI work: Availability and Cert UI's plus some normalization

### PR DESCRIPTION
**Pull request by ctaylor451**

### CODE REVIEW

File: main/templates/message_add.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+        });
+
+        if (members.length == 0) {
+            alert('Select members before paging them');
+            return;
+        }
+
+        //FIXME: what's the right way to get the current value of a checkbox
+        // this doesn't seem to work, nor does $('#id_phone').val()
+        viaEmail = $('input[name=email]').attr('checked') == 'checked' ;
```
Comment (kduncklee) [inline main/templates/message_add.html]: You can use .checked or .prop('checked')
http://api.jquery.com/prop/#prop1

File: bamru_net/urls.py [hunk 2]
```diff
@@ -30,22 +30,20 @@
...
-    path('member/', views.MemberIndexView.as_view(), name='member_index'),
+    path('member/', views.MemberListView.as_view(), name='member_list'),
     path('member/<int:pk>/', views.MemberDetailView.as_view(), name='member_detail'),
     path('member/<int:pk>/edit', views.MemberEditView.as_view(), name='member_edit'),
-    path('member/<int:pk>/certs/', views.MemberCertsView.as_view(), name='member_certs'),
-    path('member/<int:member>/certs/new', views.CertCreateView.as_view(), name='new_cert'),
-    path('member/<int:member>/certs/<int:pk>', views.CertEditView.as_view(), name='edit_cert'),
-    path('member/<int:member>/certs/<int:cert>/delete', views.CertDeleteView.as_view(), name='delete_cert'),
+    path('member/<int:pk>/certs/', views.MemberCertListView.as_view(), name='member_cert_list'),
+    path('member/<int:member>/certs/new', views.CertCreateView.as_view(), name='member_cert_new'),
```
Comment (kduncklee) [inline bamru_net/urls.py]: Please also change the names in all the tests. There are currently failing tests.

File: main/templates/message_add.html (hunk 1 above)
Comment (ctaylor451) [reply main/templates/message_add.html]: Thanks, tried lots of things but not .prop()

File: bamru_net/urls.py (hunk 2 above)
Comment (ctaylor451) [reply bamru_net/urls.py]: How do you run testing locally?  python -m test doesn't work but I'm assuming there is a simple answer

**Comment by ctaylor451** [comment]
tried running test ./manage.py test but the results are less than obvious.
LINE 1: ...."status", "message_outboundsms"."error_message", "message_o...

Let's talk in the morning

**Comment by ctaylor451** [comment]
Agreed, smaller is better in general. My goal was to get us to prototype
and then start working from PRs

**Pull request merged**